src/__experimental__/components/number/number.spec.js

```diff
@@ -114,6 +115,15 @@ describe('Number Input', () => {
       });
     });
   });
+  it('the required prop is passed to the input', () => {
+    wrapper = renderNumberInput({ label: 'required', required: true });
+
+    input = wrapper.find('input');
+    expect(input.prop('required')).toBe(true);
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.